core/pom.xml

```diff
@@ -0,0 +1,407 @@
+        <maven.site.plugin.version>3.12.1</maven.site.plugin.version>
+        <glassfishbuild.version>3.2.20.payara-p2</glassfishbuild.version>
+        <maven.antrun.plugin.version>3.1.0</maven.antrun.plugin.version>
+        <maven.assembly.plugin.version>3.4.2</maven.assembly.plugin.version>
+        <hk2.plugin.version>3.0.2</hk2.plugin.version> <!-- May need patched-->
+
+
+        <!-- build settings -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <deploy.skip>true</deploy.skip>
+        <javadoc.skip>true</javadoc.skip>
+        <opentelemetry.version>1.22.0</opentelemetry.version>
+        <opentelemetry.alpha.version>1.22.0-alpha</opentelemetry.alpha.version>
+
+    </properties>
+
+    <scm>
+        <connection>scm:git:git@github.com:payara/payara.git</connection>
+        <url>scm:git:git@github.com:payara/payara.git</url>
+        <developerConnection>scm:git:git@github.com:payara/payara.git</developerConnection>
+        <tag>payara-server-6.2023.3-SNAPSHOT</tag>
+    </scm>
+
+    <issueManagement>
+        <system>Github</system>
+        <url>https://github.com/payara/Payara/issues</url>
+    </issueManagement>
+
+    <developers>
+        <developer>
+            <name>Payara Team</name>
+            <email>info@payara.fish</email>
+            <organization>Payara Foundation</organization>
+            <organizationUrl>http://www.payara.fish</organizationUrl>
+        </developer>
+    </developers>
+
+    <licenses>
+        <license>
+            <name>CDDL + GPLv2 with classpath exception</name>
+            <url>https://raw.githubusercontent.com/payara/Payara/master/LICENSE.txt</url>
+            <distribution>repo</distribution>
+            <comments>A business-friendly OSS license</comments>
+        </license>
+    </licenses>
+
+    <repositories>
+        <repository>
+            <id>payara-nexus-artifacts</id>
+            <url>https://nexus.payara.fish/repository/payara-artifacts</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+
+    </repositories>
+
+    <pluginRepositories>
+        <pluginRepository>
+            <id>payara-nexus-artifacts</id>
+            <url>https://nexus.payara.fish/repository/payara-artifacts</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </pluginRepository>
+    </pluginRepositories>
+
+    <modules>
+        <module>core-bom</module>
+        <module>core-parent</module>
+        <!--        fish.payara.server.internal.common      glassfish-api-->
+        <module>../nucleus/common/glassfish-api</module>
```

Comment:
a) 🤢- why not just move them under core?
b) what's the distinction between core and nucleus now?

Reply:
a) Payara 5 backport simplification and minimizing merge disruption. But ultimately, we're not done with this project yet, we can still move things away later.
b) Core are the libraries lots of code (that we might want to externalize) ended up being dependent upon. There is no aim of building shippable distribution out of core.